src/log/log_slot.c

```diff
@@ -71,7 +71,8 @@ __wt_log_slot_activate(WT_SESSION_IMPL *session, WT_LOGSLOT *slot)
      * are reset when the slot is freed.  See log_slot_free.
      */
     slot->slot_unbuffered = 0;
-    slot->slot_start_lsn = slot->slot_end_lsn = log->alloc_lsn;
+    WT_ASSIGN_LSN(&slot->slot_start_lsn, &log->alloc_lsn);
+    WT_ASSIGN_LSN(&slot->slot_end_lsn, &slot->slot_start_lsn);
```

Comment:
I think this would read more clearly if the second call to `WT_ASSIGN_LSN` also had `&log->alloc_lsn` as the second arg.

Reply:
nvm, I see you've already been down this road.